README.md

```diff
@@ -281,3 +281,38 @@ Run the command below to generate a list of infrastructure values. These values
 ```
 
 This command will output a set of key-value pairs, extracted from the infrastructure setup, that can be used as inputs in the domino.yaml configuration file.
+
+
+## Domino Backups
+To make our backups safe we could use cross account backups (Accounts under same AWS Organization), in case you want to apply this you can use the following [module](https://github.com/dominodatalab/terraform-aws-domino-backup) to enabled this and the following steps:
+
+1. Define another provider for the backup account in `main.tf` for infra module.
+
+Location
+```bash
+domino-deploy
+├── terraform
+│   ├── infra
+│   │   ├── main.tf
+```
+
+Content
+```
+provider "aws" {
+  alias   = "domino-backup"
+  region  = <<Backup Account Region>>
+  profile = <<Profile Account Here>>
```

Comment:
Maybe leave `profile` out since technically it could be any credential method?

Reply:
remove it